sys-auth/fprintd/fprintd-0.8.1-r3.ebuild

```diff
@@ -13,25 +13,28 @@ SRC_URI="https://cgit.freedesktop.org/libfprint/${PN}/snapshot/${MY_PV}.tar.bz2
 LICENSE="GPL-2"
 SLOT="0"
 KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~ia64 ~ppc ~ppc64 ~sparc ~x86"
-IUSE="doc pam static-libs"
+IUSE="doc pam"
 
 RDEPEND="
 	dev-libs/dbus-glib
 	dev-libs/glib:2
 	sys-auth/libfprint:0
 	sys-auth/polkit
-	pam? ( sys-libs/pam )
-"
+	pam? ( sys-libs/pam )"
 DEPEND="${RDEPEND}"
 BDEPEND="
 	dev-libs/dbus-glib
 	dev-util/gtk-doc
 	dev-util/gtk-doc-am
 	dev-util/intltool
-	doc? ( dev-libs/libxml2 dev-libs/libxslt )
-"
+	doc? (
+		dev-libs/libxml2
+		dev-libs/libxslt
+	)"
 
-S=${WORKDIR}/${MY_PV}
+PATCHES=( "${FILESDIR}"/${P}-fno-common.patch )
+
+S="${WORKDIR}"/${MY_PV}
```

Comment:
swap these, PATCHES definition should always come after $S

Reply:
Oh, that's right. Done